src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -271,6 +274,12 @@ protected override void OnApplyTemplate()
 				_rootGrid.Tapped += OnRootGridTapped;
 			}
 
+			_controlPanelGrid = this.GetTemplateChild(ControlPanelGridName) as Grid;
+			if (_controlPanelGrid != null)
+			{
+				_controlPanelGrid.Tapped -= OnPaneGridTapped;
+				_controlPanelGrid.Tapped += OnPaneGridTapped;
```

Comment:
This is likely going to leak. We should be hooking on any event in OnApplyTemplate, but rather in Loaded/Unloaded.

Reply:
done.